types/node/test/process.ts

```diff
@@ -129,3 +129,26 @@ process.env.TZ = 'test';
 {
     const arch: NodeJS.Architecture = process.arch;
 }
+
+{
+    // $ExpectError
+    process.getgid();
```

Comment:
Potentially we could also test to call all the functions using optional chaining operator, to make sure that the functions still exists and are typed properly?

```suggestion
    // $ExpectError
    process.getgid();
    process.getgid?.();
```

Reply:
Added a bunch more tests